README.md

```diff
@@ -245,10 +245,104 @@ If you experienced a compatibility issue with any of the following plugins, plea
 - [mini-css-extract-plugin](https://github.com/webpack-contrib/mini-css-extract-plugin)
 - [HtmlWebpackPlugin](https://github.com/jantimon/html-webpack-plugin)
 
+### options.contentScript.experimental_output
+
+**This is an experimental API. API might change at any time. Please provide feedback!**
+
+#### TLDR: How to use this
+
+##### string
+
+If you don't strictly rely on [run_at](https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/manifest.json/content_scripts#run_at),
+set it as the following
+
+```js
+export default {
+  entry: {
+    myContentScript: 'src/contentScript.ts',
+  },
+  // ...
+  plugins: [
+    // ...
+    new WebExtensionPlugin({
+      // ...
+      contentScript: {
+        experimental_output: {
+          myContentScript: 'cs.js'
```

Comment:
so `cs.js` refers to the output file for content_scripts within the root of the output path? like `dist/cs.js`?

Reply:
Yes, it will be an extra output file.